aspnetcore/fundamentals/servers/kestrel/endpoints/includes/endpoints5-7.md

```diff
@@ -108,6 +108,8 @@ Configure Kestrel to use HTTPS.
 
 In production, HTTPS must be explicitly configured. At a minimum, a default certificate must be provided.
 
+If certificates are being read from disk (as opposed to a [Windows Certificate Store](https://learn.microsoft.com/windows-hardware/drivers/install/certificate-stores)), the containng directory must have appropriate permissions to prevent unauthorized access.
```

Comment:
```suggestion
If certificates are being read from disk (as opposed to a [Windows Certificate Store](https://learn.microsoft.com/windows-hardware/drivers/install/certificate-stores)), the containing directory must have appropriate permissions to prevent unauthorized access.
```